contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.0.0--3.1.0.sql

```diff
@@ -0,0 +1,54 @@
+-- complain if script is sourced in psql, rather than via ALTER EXTENSION
+\echo Use "ALTER EXTENSION ""babelfishpg_common"" UPDATE TO '3.1.0'" to load this file. \quit
```

Comment:
Please add `2.3.0--2.4.0` and `2.4.0--3.0.0` upgrade scripts.

Reply:
Fixed in the latest commit